lte/gateway/python/magma/mobilityd/ip_address_man.py

```diff
@@ -122,7 +123,8 @@ def __init__(self,
         """
 
         persist_to_redis = config.get('persist_to_redis', True)
-        redis_port = config.get('redis_port', 6379)
+        redis_port = get_service_config_value('redis', 'port', 6379)
```

Comment:
nit: I'd prefer to get these values by getting the redis service config on the mobilityd/main and then add it as parameter to the rpc servicer, then you can pass the object as redis_config, and unpack the values here